### PR TITLE
Added script for calculating RotD100

### PR DESCRIPTION
**Pull request by stansays**
Also commented out openquake imports in `sm_utils.py`
It doesn't seem like numba would work with the unsupported constructs such as classes or dicts needed for response spectrum:
https://numba.pydata.org/numba-doc/dev/reference/pysupported.html#:~:text=Unsupported%20constructs

### CODE REVIEW

**Comment by stansays** [comment]
Added instead multiprocessing pool for parallelizing the acceleration time histories

File: src/RotD100.py [hunk 1]
```diff
@@ -11,25 +12,37 @@ def parse_record_txt(file):
...
 
-
-periods = np.array([
-    0.01, 0.02, 0.03, 0.04, 0.05, 0.075, 0.1, 0.11, 0.12, 0.13, 0.14, 0.15,
-    0.16, 0.17, 0.18, 0.19, 0.20, 0.22, 0.24, 0.26, 0.28, 0.30, 0.32, 0.34,
-    0.36, 0.38, 0.40, 0.42, 0.44, 0.46, 0.48, 0.5, 0.55, 0.6, 0.65, 0.7, 0.75,
-    0.8, 0.85, 0.9, 0.95, 1.0, 1.1, 1.2, 1.3, 1.4, 1.5, 1.6, 1.7, 1.8, 1.9,
-    2.0, 2.2, 2.4, 2.6, 2.8, 3.0, 3.2, 3.4, 3.6, 3.8, 4.0, 4.2, 4.4, 4.6, 4.8,
-    5.0, 5.5, 6.0, 6.5, 7.0, 7.5, 8.0, 8.5, 9.0, 9.5, 10.0
-],
```
Comment (ftbernales) [inline src/RotD100.py]: This period list is HUGE. Did you %time this with the mp feature?

File: src/RotD100.py [hunk 2]
```diff
@@ -11,25 +12,37 @@ def parse_record_txt(file):
...
-                   dtype=float)
-
-cwd = os.path.abspath('../data/input/ATH/')
-text_files = [f for f in os.listdir(cwd) if f.endswith('.txt')]
-for i in range(0, len(text_files), 2):
+def compute_rotd100(acc_record):
+    x = acc_record[0]
+    y = acc_record[1]
+    periods = np.array([
+        0.01, 0.02, 0.03, 0.04, 0.05, 0.075, 0.1, 0.11, 0.12, 0.13, 0.14, 0.15,
```
Comment (ftbernales) [inline src/RotD100.py:19]: This period list is HUGE. Did you %time this with the mp feature?

**Comment by ftbernales** [comment]
@stansays Apologies for not noticing this ahead of time. You can try out my [Initial commit for matching_assessment](https://github.com/stansays/Deepsoil-Compile/pull/1/commits/6277575e4987ddc4f59d2844a3d90ff4728e2435) to check out my grand plan for this feature. 

But thanks for adding the mp pool. That should still work in the `main()` scripting function, right?

**Comment by ftbernales** [comment]
Done merging the files here to [matching_assessment/](https://github.com/ftbernales/Deepsoil-Compile/tree/matching_assessment)

**Pull request closed without merging**